lib/Twig/Extension/Escaper.php

```diff
@@ -78,13 +85,13 @@ public function setDefaultStrategy($defaultStrategy)
      *
      * @param string $filename The template "filename"
      *
-     * @return string The default strategy to use for the template
+     * @return string|false The default strategy to use for the template
      */
     public function getDefaultStrategy($filename)
     {
         // disable string callables to avoid calling a function named html or js,
         // or any other upcoming escaping strategy
-        if (!is_string($this->defaultStrategy) && is_callable($this->defaultStrategy)) {
+        if (is_array($this->defaultStrategy)) {
```

Comment:
this is wrong. You can have other callables than arrays (invokables objects are allowed)